Hogajama/hogajama-services/pom.xml

```diff
@@ -12,24 +12,28 @@
 	<artifactId>hogajama-services</artifactId>
 	<packaging>jar</packaging>
 
-	<dependencies>
-		<dependency>
-			<groupId>com.gepardec.hogajama</groupId>
-			<artifactId>hogajama-domain</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.mongodb.morphia</groupId>
-			<artifactId>morphia</artifactId>
-		</dependency>
+  <dependencies>
+    <dependency>
+      <groupId>com.gepardec.hogajama</groupId>
+      <artifactId>hogajama-domain</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.mongodb.morphia</groupId>
+      <artifactId>morphia</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>5.10.1</version>
```

Comment:
Is there a reason, why the versions is not managed in the dependency-management section of the parent pom?